packages/components/src/internal/components/labels/LabelsConfigurationPanel.tsx

```diff
@@ -79,16 +96,37 @@ const normalizeValues = (template: LabelTemplate): LabelTemplate => {
         description: template.description,
         path: template.path?.trim(),
         rowId: template.rowId,
+        container: template.container,
     });
 };
 
+const canBeDefault = (template: LabelTemplate): boolean => {
+    const currentContainer = getServerContext().container;
+    return (
+        !template || // New template
+        currentContainer.parentId === template.container || // Template is from the project level
+        currentContainer.id === template?.container // Template is from this project
```

Comment:
do we need the null check for `template?.container` here? if so, it is likely needed in the line about (but I don't think it is needed for either).

Reply:
The `!template` should cover it. removed.